Miniforge3/construct.yaml

```diff
@@ -49,5 +49,11 @@ specs:
   - pip
   - miniforge_console_shortcut 1.*  # [win]
 
+{% if name.startswith("Mambaforge") %}
 pre_install: mambaforge_deprecation.sh  # [unix]
 pre_install: mambaforge_deprecation.bat  # [win]
+{% endif %}
```

Comment:
Might want to come this out if you don't want the deprecations in this release @hmaarrfk

Reply:
It would be good to have the message in the release, but not the "forced inability to use it".

Then we can point people to the release.

Could i comment out:
https://github.com/jaimergp/miniforge/blob/system-requirements-and-conda-24.7.1/Miniforge3/mambaforge_deprecation.sh#L12
and the associated lines in the bat file and we can merge this for a release?

Reply:
suggestion:    https://github.com/conda-forge/miniforge/pull/634

Reply:
i just merged the other.